src/appleseed/renderer/modeling/light/maxspotlight.cpp

```diff
@@ -109,10 +109,10 @@ namespace
             const double outer_half_angle = deg_to_rad(m_params.get_required<double>("outer_angle", 30.0) / 2.0);
             const double tilt_angle = deg_to_rad(m_params.get_optional<double>("tilt_angle", 0.0));
 
-            m_cos_inner_half_angle = cos(inner_half_angle);
-            m_cos_outer_half_angle = cos(outer_half_angle);
-            m_rcp_screen_half_size = 1.0 / tan(outer_half_angle);
-            m_up = Vector3d(sin(tilt_angle), cos(tilt_angle), 0.0);
+            m_cos_inner_half_angle = std::cos(inner_half_angle);
+            m_cos_outer_half_angle = std::cos(outer_half_angle);
+            m_rcp_screen_half_size = 1.0 / std::tan(outer_half_angle);
+            m_up = Vector3d(sin(tilt_angle), std::cos(tilt_angle), 0.0);
```

Comment:
`std::sin()`

Reply:
Ah, I had a script doing the work, but only considered ` sin` but not `(sin`. :D
I will modify and rerun.